### ISSUE TITLE
ZipNeRF是否具有场景限制？

### ISSUE THREAD

**Issue by s1eeveW** [SuLvXiangXin/zipnerf-pytorch#104]
I used the 'Tank and Temple dataset'. For the results, some scenes are slightly better than the 3DGS results (e.g., Truck and Caterpillar); however, some scenes' metrics are far worse than 3DGS (e.g., Train and Courthouse).

Comparison of poor rendering results:
3DGS - 
![00096](https://github.com/SuLvXiangXin/zipnerf-pytorch/assets/64772045/b172bb5a-555b-4e92-a4e7-c8e2c0be320f)

ZIPNeRF
![color_012](https://github.com/SuLvXiangXin/zipnerf-pytorch/assets/64772045/352c5bde-302f-4071-a321-6c7fc467a264)

3DGS - 
![00016](https://github.com/SuLvXiangXin/zipnerf-pytorch/assets/64772045/e1495cd9-ba19-4d28-b8ec-1f92d2bc6161)

ZIPNeRF - 
![color_002](https://github.com/SuLvXiangXin/zipnerf-pytorch/assets/64772045/da469db4-fdf3-4888-a75b-d8468024b469)

My input (for Train as an example):
CUDA_VISIBLE_DEVICES=0 python train.py --gin_configs=configs/360.gin --gin_bindings="Config.data_dir = '/NeRF_Images/Train'" --gin_bindings="Config.exp_name = '/Output/Train'" --gin_bindings="Config.factor = 1"

CUDA_VISIBLE_DEVICES=0 python eval.py --gin_configs=configs/360.gin --gin_bindings="Config.data_dir = '/NeRF_Images/Train'" --gin_bindings="Config.exp_name = '/Output/Train'" --gin_bindings="Config.factor = 1"

Changes in configs.py:
batch_size: int = 4096
render_chunk_size: int = 4096
max_steps: int = 30000

Could anyone help me find the reason?

**Comment by ichsan2895** [comment]
Sorry for super late respond.

AFAIK

`ZipNerf` in `Nerfstudio` has different hyperparameter than original `ZipNerf`

Original Zipnerf has [Config.batch_size = 65536](https://github.com/jonbarron/camp_zipnerf/blob/8e6d57e3aee34235faf3ef99decca0994efe66c9/configs/zipnerf/360.gin#L8C3-L8C28)

ZipNerf implementation in Nerfstudio has lower value. It has [8192 rays/batch](https://github.com/SuLvXiangXin/zipnerf-pytorch/blob/af86ea6340b9be6b90ea40f66c0c02484dfc7302/zipnerf_ns/zipnerf_config.py#L38

So, for fair benchmark, change ns-train zipnerf with batch size = 65536 rays/batch.

**Comment by s1eeveW** [comment]
Thanks for your reply!

How do we know the ZipNeRF model is converged?

I adjusted the batch size to 32,768 and set the max_step to 25,000, resulting in the following loss curve and validation results every 500 steps on Tank and Temples dataset - truck environment:
![total_loss_curve](https://github.com/user-attachments/assets/bdf6580e-67ad-4a99-bed8-a271e0be7af1)
![valid_psnr_curve](https://github.com/user-attachments/assets/890f5737-6c6a-451c-8756-5a1f5aa935c8)

**Comment by s1eeveW** [comment]
@ichsan2895
